internal/worker/save_data.go

```diff
@@ -90,3 +99,16 @@ func SaveFileWritesData(ctx context.Context, pkg *pkgmanager.Pkg, dest ResultSto
 
 	return nil
 }
+
+// SaveAnalyzedPackage saves the analyzed package from static and dynamic analysis to the analyzed packages bucket in the ResultStores
+func SaveAnalyzedPackage(ctx context.Context, pkg *pkgmanager.Pkg, dest ResultStores) error {
+	if pkg.IsLocal() {
+		return nil
+	}
+
+	if err := dest.AnalyzedPackage.SaveAnalyzedPackage(ctx, pkg.Manager(), pkg); err != nil {
+		return fmt.Errorf("failed to upload analyzed package to blobstore = %w", err)
```

Comment:
nit: use error format consistent with other error messages in this file 
(i.e. `fmt.Errorf("failed to upload analyzed package to %s: %w", dest.AnalyzedPackage, err)`)